### DISCUSSION TITLE
Large file download on Swagger UI with FileResponse / StreamingResponse. Chrome out of memory issue.

### DISCUSSION THREAD

**Discussion by moono** [fastapi/fastapi#6464] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI
from fastapi.responses import StreamingResponse, FileResponse


app = FastAPI()


@app.get("/small_file_please")
async def get_small_file():    
    fn = "small_864K.png"
    response = FileResponse(fn, media_type="application/octet-stream", filename=fn)
    return response


@app.get("/medium_file_please")
async def get_medium_file():    
    fn = "medium_995M.zip"
    response = FileResponse(fn, media_type="application/octet-stream", filename=fn)
    return response


@app.get("/large_file_please")
async def get_large_file():    
    fn = "large_1.5G.zip"
    response = FileResponse(fn, media_type="application/octet-stream", filename=fn)
    return response
```

### Description

* I was trying to create some endpoint to download file using FastAPI.
* It worked very good for small files, but as soon as I tried with some large file, the chrome browser just crashes upon loading completes. Showing following message,

![image](https://user-images.githubusercontent.com/6761398/145156720-bbaaaaaf-d98d-406e-8e55-d6feb3114ca0.png)

* `StreamingResponse` or `FileResponse` has same situation.
* In my case, up to around 1G was okay but when I tried with more than 1.4G file it crashes.
* At first I looked up google to find a way to increase memory limit with `--max_old_space_size=4096` but no luck.
* And looked for fastapi related posts but couldn't find it as well.
* So I was wondering If anyone had simillar issue and guide me to right direction?

### Operating System

Linux

### Operating System Details

FastAPI is running on Ubuntu 18.04. (Inside docker container)
Tested chrome browser was on Windows 10.

### FastAPI Version

0.70.0

### Python Version

3.8.10

**Comment by moono** [comment]
I can confirm that with `curl` command the api works correctly, so it is not `FastAPI` issue. 
Sorry about interrupt!!

**Reply by Dogrtt** [reply]
But how to handle it?
There more, a few weeks ago I was able to download 4GB file via StreamResponse, but now I'm facing OOM page on 1.3Gb file downloading.
"Invoke-WebRequest" working with this endpoin, so it's not a problem of API by itself, but developers and testers wanna use swagger!

**Reply by YuriiMotov** [reply]
I think you can try searching this on Swagger UI repository or asking question there
